### DISCUSSION TITLE
Release 3.0.0-alpha3

### DISCUSSION THREAD

**Discussion by meatballhat** [urfave/cli#1739] [Announcements]
## What's Changed
* Fix some issues in bash autocompletion by @MrNaif2018 in https://github.com/urfave/cli/pull/1674
* Adjust sub-command template to properly output sub-sub-command placeholder. by @aryounce in https://github.com/urfave/cli/pull/1683
* Clean up some documentation by @docwhat in https://github.com/urfave/cli/pull/1688
* Shift tested Go versions by @meatballhat in https://github.com/urfave/cli/pull/1691
* Updated badges for v3 by @meatballhat in https://github.com/urfave/cli/pull/1665
* Fix linting issues by @skelouse in https://github.com/urfave/cli/pull/1694
* v3 Add integration with golangci-lint by @skelouse in https://github.com/urfave/cli/pull/1697
* Add makezero linter by @avorima in https://github.com/urfave/cli/pull/1705
* fix some broken links in docs by @doooks in https://github.com/urfave/cli/pull/1698
* Feat: Added tabular markdown writer by @tarampampam in https://github.com/urfave/cli/pull/1722

## New Contributors
* @aryounce made their first contribution in https://github.com/urfave/cli/pull/1683
* @docwhat made their first contribution in https://github.com/urfave/cli/pull/1688
* @doooks made their first contribution in https://github.com/urfave/cli/pull/1698
* @tarampampam made their first contribution in https://github.com/urfave/cli/pull/1722

**Full Changelog**: https://github.com/urfave/cli/compare/v3.0.0-alpha2...v3.0.0-alpha3

<hr /><em>This discussion was created from the release <a href='https://github.com/urfave/cli/releases/tag/v3.0.0-alpha3'>Release 3.0.0-alpha3</a>.</em>

**Comment by Integralist** [comment]
👋 is there a milestone I can track for v3?

Also where can I find the docs for v3? 

https://github.com/urfave/cli/tree/v3.0.0-alpha4/docs/v3 looks to be empty so I presume I have to use the pkg.go.dev reference instead.

Thanks.
